lib/instagram.js

```diff
@@ -373,6 +373,27 @@ var instagram = function(spec, my) {
   };
 
   /**
+   * Retrieves information about the given user
+   * @param id string the user id
+   * @param cb function (err, user, remaining, limit);
+   */
+  user_self = function(cb) {
```

Comment:
the variable `user_self` isn't declared above with the other variable declarations. +1 though!